### PR TITLE
[plugin.video.catchuptvandmore] 0.2.10

### PR DESCRIPTION
**Pull request by SylvainCecchetto**
### Description

[Version 0.2.10]
[Fix] Catch-up TV - France TV (FR) - Case Fort Boyard
[Add] Live TV - IDF 1 (FR)
[Improvements] Use external tzlocal module
[Fix] Catch-up TV - Arte (WO) - Fix Description
[Fix] Guide TV (BE) - Need to rework the guide TV
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.catchuptvandmore/resources/lib/channels/wo/arte.py [hunk 1]
```diff
@@ -231,7 +231,9 @@ def list_programs(plugin, item_id, sub_category_code_name, sub_category_url,
                                 'landscape']['resolutions']:
                             video_image = video_image_datas['url']
                     video_duration = program_datas["duration"]
-                    video_plot = program_datas["description"]
+                    video_plot = ''
```
Comment (basrieter) [inline plugin.video.catchuptvandmore/resources/lib/channels/wo/arte.py]: Why not use a `program_datas.get("description", "")`

File: plugin.video.catchuptvandmore/resources/lib/channels/wo/arte.py [hunk 2]
```diff
@@ -306,7 +308,9 @@ def list_videos_sub_category(plugin, item_id, sub_category_url,
                         'landscape']['resolutions']:
                     video_image = video_image_datas['url']
             video_duration = video_datas["duration"]
-            video_plot = video_datas["description"]
+            video_plot = ''
```
Comment (basrieter) [inline plugin.video.catchuptvandmore/resources/lib/channels/wo/arte.py]: Why not use a `video_datas.get("description", "")`

File: plugin.video.catchuptvandmore/resources/lib/channels/wo/arte.py [hunk 3]
```diff
@@ -350,7 +354,9 @@ def list_videos_sub_category(plugin, item_id, sub_category_url,
                                 'landscape']['resolutions']:
                             video_image = video_image_datas['url']
                     video_duration = video_datas["duration"]
-                    video_plot = video_datas["description"]
+                    video_plot = ''
```
Comment (basrieter) [inline plugin.video.catchuptvandmore/resources/lib/channels/wo/arte.py]: Why not use a `video_datas.get("description", "")`

File: plugin.video.catchuptvandmore/resources/lib/channels/wo/arte.py [hunk 4]
```diff
@@ -435,7 +441,9 @@ def list_videos_program_concert(plugin, item_id, program_url,
                         'landscape']['resolutions']:
                     video_image = video_image_datas['url']
             video_duration = video_datas["duration"]
-            video_plot = video_datas["description"]
+            video_plot = ''
```
Comment (basrieter) [inline plugin.video.catchuptvandmore/resources/lib/channels/wo/arte.py]: Same here

File: plugin.video.catchuptvandmore/resources/lib/channels/wo/arte.py [hunk 5]
```diff
@@ -477,7 +485,9 @@ def list_videos_program_concert(plugin, item_id, program_url,
...
```
Comment (basrieter) [inline plugin.video.catchuptvandmore/resources/lib/channels/wo/arte.py]: Same here

File: plugin.video.catchuptvandmore/resources/language/resource.language.en_gb/strings.po [hunk 6]
```diff
@@ -371,6 +371,8 @@ msgstr ""
 
 
 
+
```
Comment (basrieter) [inline plugin.video.catchuptvandmore/resources/language/resource.language.en_gb/strings.po]: Why all the lines?

**Comment by SylvainCecchetto** [comment]
Thank you for your review. I made the changes.
However Travis seems broken :-/

**Pull request merged**